src/include/ipc/header.h

```diff
@@ -170,7 +170,7 @@
 #define SOF_IPC_MESSAGE_ID(x)			((x) & 0xffff)
 
 /** Maximum message size for mailbox Tx/Rx */
-#define SOF_IPC_MSG_MAX_SIZE			384
+#define SOF_IPC_MSG_MAX_SIZE			768
```

Comment:
@abonislawski my understanding after recent discussions is that we are moving to 2kB.

Reply:
@lgirdwood To move to 2kB we have to rework how msgs are allocated. @abonislawski will work on that, but for now we would like to use as much as we can without these changes.

Reply:
@lgirdwood just like @jajanusz said, please merge

Reply:
@plbossart hardly to say but atm Im busy with other tasks

Reply:
@plbossart It's not 5-minutes-task. Msg allocation and handling code would need to be reworked and we have more urgent tasks now. We want to increase it to 768 atm, cos without it we stall another feature. I know it's inconvenient for you, but we can't help much with that now. We would greatly appreciate your help with approving this one.

Reply:
Humm, what happens if I don't? The mux would be stalled, but what is the impact? Put differently, who needs the mux support from the kernel level in the coming weeks?